samples/api/oit_linked_lists/oit_linked_lists.cpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2023, Google
+/* Copyright (c) 2024, Google
```

Comment:
You shouldn't need to update copyright if the file hasn't been changed in this PR

Reply:
Fixed